README.md

```diff
@@ -1,9 +1,22 @@
-# INSPIRE XSD application schemas
+# INSPIRE XML schemas
 
-The purpose of this repository is to manage the changes to the INSPIRE XSD application schemas (hereafter INSPIRE schemas).
+The purpose of this repository is to manage the changes to the INSPIRE XML schemas.
 
-The official location of the INSPIRE schemas — to be used within the INSPIRE resources — is, and will remain, https://inspire.ec.europa.eu/schemas. This repository, which is setup within [Action 2.3](https://webgate.ec.europa.eu/fpfis/wikis/display/InspireMIG/Action+2.3+Simplification+of+INSPIRE+implementation) of the [INSPIRE MIWP 2021-2024](https://webgate.ec.europa.eu/fpfis/wikis/display/InspireMIG/INSPIRE+work+programme+2021-24), is only a means to facilitate the change management process of the INSPIRE schemas.
+The INSPIRE XML schemas can be categorized in two categories:
 
-Change proposals to the current version of the schemas can be proposed and discussed in the issue tracker of this repository. These change proposals are then subject to the specific governance and release process, whose workflow and actors involved are detailed in [this document](https://github.com/INSPIRE-MIF/application-schemas/blob/main/governance-release-process/process.md).
+* GML application schemas, derived from the UML appliation schemas
```

Comment:
there is a typo here in the word 'application'

Reply:
Fixed.